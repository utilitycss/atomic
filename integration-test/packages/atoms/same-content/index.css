.bar {
	color: blue;
}

.foo_bar {
	color: blue;
}
