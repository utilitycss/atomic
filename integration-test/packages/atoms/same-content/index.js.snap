module.exports = {
  bar: "EV62sS",
  foo_bar: "JYUOso"
};